runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcCommitWorkStream.java

```diff
@@ -57,7 +60,9 @@ final class GrpcCommitWorkStream
 
   private static final long HEARTBEAT_REQUEST_ID = Long.MAX_VALUE;
 
-  private final ConcurrentMap<Long, PendingRequest> pending;
+  private final ConcurrentMap<Long, KV<CommitWorkPhysicalStreamHandler, PendingRequest>> pending =
```

Comment:
It looks like we move all pending requests to the new steam on onNewStream, it is not clear to me if there'll be multiple CommitWorkPhysicalStreamHandlers be in use at the same time. Will it change in the following PRs?